### PR TITLE
add docs for python bridge

### PR DESCRIPTION
**Pull request by kmaehashi**
Docs for https://github.com/jubatus/jubatus/pull/1208.

### CODE REVIEW

File: source/ja/fv_convert.rst [hunk 1]
```diff
@@ -932,4 +935,200 @@ binary_typesで指定できる。`OpenCV <https://github.com/opencv>`_ を利用
...
+        }
+      },
+      "string_rules": [
+        { "key": "*", "type": "stem_sentence", "sample_weight": "tf", "global_weight": "bin" }
+      ]
+
+``word_splitter`` インタフェース
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+クラスにインスタンスメソッド ``split`` (1 引数) を定義する必要がある。
```
Comment (shiodat) [inline source/ja/fv_convert.rst]: `word_splitter` interface needs to implement not `extract` but `split` method and it is bit confusing.
If you agree, we should write python implementation example like:

```
class MyWordSplitter(object):
    @classmethod
    def create(cls, param):
         return cls()

    def split(self, text):
        return text.split(',')
```

File: source/ja/fv_convert.rst [hunk 2]
```diff
@@ -932,4 +935,200 @@ binary_typesで指定できる。`OpenCV <https://github.com/opencv>`_ を利用
...
+          "method": "dynamic",
+          "path": "libpython_bridge.so",
+          "function": "binary_feature",
+          "module": "binary_length",
+          "class": "BinaryLengthExtractor"
+        }
+      }
+
+上記に加えて、Pythonクラスに渡す任意のパラメタを指定することができる。
+パラメタのキーとバリューは共に文字列型である必要がある。
```
Comment (shiodat) [inline source/ja/fv_convert.rst]: In this example, other parameters are not exist and users may think where is the parameter.
I think `multiplier_by_3` example is appropriate here.

File: source/ja/fv_convert.rst [hunk 3]
```diff
@@ -932,4 +935,200 @@ binary_typesで指定できる。`OpenCV <https://github.com/opencv>`_ を利用
...
+      def create(cls, param):
+          return cls()
+
+      def extract(self, key, value):
+          return [(u'key1', 1.0), (u'key2', 2.0)]
+
+Pythonクラスに対する要件を以下に示す。
+
+* クラスは ``create`` クラスメソッドを実装する必要がある。
+  引数 ``params`` は、設定ファイルで指定されたパラメタ (``dict`` 型) である。
```
Comment (shiodat) [inline source/ja/fv_convert.rst]: In the example, `create(cls, param)` is used,  and here `params` is used, which is correct?

File: source/ja/fv_convert.rst (hunk 2 above)
Comment (kmaehashi) [inline source/ja/fv_convert.rst]: Agree, fixed as suggested.

File: source/ja/fv_convert.rst (hunk 3 above)
Comment (kmaehashi) [inline source/ja/fv_convert.rst]: Fixed to `param`.
It's just a method parameter, so any name is fine.

File: source/ja/fv_convert.rst (hunk 1 above)
Comment (kmaehashi) [inline source/ja/fv_convert.rst]: I added example method signatures for each interface!

Comment (shiodat) [inline source/ja/fv_convert.rst]: Thanks!

**Comment by shiodat** [comment]
Good :+1: 

I found an build error and after fixing it, I'm going to merge this PR.
```
pickling environment... /home/daats/jubatus/website/source/ja/fv_convert.rst:910: ERROR: Unexpected indentation.
```

**Comment by shiodat** [comment]
LGTM :+1:

**Comment by kmaehashi** [comment]
I found a small error in example method signature, directly fixed via 1298c2537d8cfc9aa6089b7b7ff31523a7288492

**Pull request merged**